src/core.c/CompUnit/RepositoryRegistry.pm6

```diff
@@ -149,7 +149,7 @@ class CompUnit::RepositoryRegistry {
         my str $prefix = nqp::ifnull(
           nqp::atkey($ENV,'RAKUDO_PREFIX'),
           nqp::gethllsym('default', 'SysConfig').rakudo-home()
-        );
+        ).IO.absolute.IO.resolve.Str;
```

Comment:
Calling `.IO` multiple times seems redundant. I think `.IO.resolve.absolute` is equivalent

Reply:
No, it isn't, unfortunately. I need to recover when it wasn't interchangeable, but this order was experimentally proved.

I'd rather have `absolute` return `IO` too. Beats me why does it return `Str` together with `relative`.

Reply:
`.Str` returns the form originally supplied to the constructor so to speak. `.absolute` returns the absolutified string form, and `.relative` the relative form. There isn't really a need for `.absolute` to return an `IO` object because that object is supposed to be aware of the context needed to access the file regardless of it it was supplied a relative or absolute path (since it knows the cwd).